### DISCUSSION TITLE
auto install script

### DISCUSSION THREAD

**Discussion by hornethosting** [pelican-dev/panel#71] [Ideas]
if you could would love this https://github.com/pterodactyl-installer/pterodactyl-installer

**Comment by hornethosting** [comment]
not very important would be nice tho

**Comment by WolfoTheWolf** [comment]
They are already planning it last I heard, one line installer.

**Reply by abhicommands** [reply]
this would reduce the number of support tickets created by almost 99%

**Comment by lancepioch** [comment]
Thanks, we are actively working on bettering the installation (and update) methods!
